pkg/scheduler/framework/plugins/clusteraffinity/state.go

```diff
@@ -0,0 +1,42 @@
+package clusteraffinity
+
+import (
+	fleetv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+type pluginState struct {
+	// requiredAffinityTerms is a list of processed version of required cluster affinity terms.
+	requiredAffinityTerms []AffinityTerm
+
+	// preferredAffinityTerms is a list of processed version of preferred cluster affinity terms.
+	preferredAffinityTerms *PreferredAffinityTerms
```

Comment:
A nit here: for consistency here it should either all use aliases, or all use the plain types.